e2e_samples/parking_sensors_synapse/.devcontainer/Dockerfile

```diff
@@ -2,12 +2,36 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See LICENSE in the project root for license information.
 #-----------------------------------------------------------------------------------------
-FROM python:3.7-stretch
+# FROM python:3.7-stretch
 
-# Configure apt
-ENV DEBIAN_FRONTEND=noninteractive
-RUN apt-get update \
-    && apt-get -y install --no-install-recommends apt-utils 2>&1
+# # Configure apt
+# ENV DEBIAN_FRONTEND=noninteractive
+# RUN apt-get update \
+#     && apt-get -y install --no-install-recommends apt-utils 2>&1
+
+FROM ubuntu:16.04
```

Comment:
We may want to think about using a more up to date version of ubuntu like 20.04 LTS